ReCaptchaFrontendUi/etc/frontend/di.xml

```diff
@@ -11,8 +11,4 @@
                 type="Magento\ReCaptchaFrontendUi\Model\CaptchaTypeResolver"/>
     <preference for="Magento\ReCaptchaUi\Model\ErrorMessageConfigInterface"
                 type="Magento\ReCaptchaFrontendUi\Model\ErrorMessageConfig"/>
-    <type name="Magento\Framework\View\Asset\Minification">
```

Comment:
Why was this removed?

Reply:
https://*/recaptcha/api.js is not catchable by the Magento minifier as the url is set dynamically in js

Reply:
@nathanjosiah